src/context.tsx

```diff
@@ -15,173 +26,123 @@ export function getEmptyFilters(): Filters {
+    minDate,
+    maxDate,
   },
   language: LanguageType.english,
-  updatedAt: '',
+  updatedAt: "",
   showCookieBanner: false,
-  showAnalyzePopup: true,
-  countries : [],
+  countries: [],
   showEstimatePins: true,
-  showCountryHover:true,
+  showCountryHover: true,
 };
 
 const reducer = (state: State, action: Record<string, any>): State => {
   switch (action.type) {
     case "SHOW_COUNTRY_HOVER":
       return {
         ...state,
-        showCountryHover: true
+        showCountryHover: true,
       };
     case "HIDE_COUNTRY_HOVER":
       return {
         ...state,
-        showCountryHover: false
+        showCountryHover: false,
       };
     case "TOGGLE_ESTIMATE_PINS":
       return {
         ...state,
-        showEstimatePins: !state.showEstimatePins
+        showEstimatePins: !state.showEstimatePins,
       };
     case "CLOSE_COOKIE_BANNER":
       return {
         ...state,
-        showCookieBanner: false
+        showCookieBanner: false,
       };
     case "OPEN_COOKIE_BANNER":
       return {
         ...state,
-        showCookieBanner: true
-      };
-    case "OPEN_ANALYZE_POPUP":
-      return {
-        ...state,
-        showAnalyzePopup: true
-      };
-    case "CLOSE_ANALYZE_POPUP":
-      return {
-        ...state,
-        showAnalyzePopup: false
+        showCookieBanner: true,
       };
     case "UPDATE_ESTIMATE_PREVALENCES": {
       const { pageStateEnum, estimateGradePrevalences } = action.payload;
       const newState = { ...state };
       const pageState = newState[pageStateEnum as keyof State] as PageState;
       pageState.estimateGradePrevalences = estimateGradePrevalences;
       return newState;
-    };
+    }
     case "UPDATE_COUNTRIES_JSON":
       return {
         ...state,
-        countries: action.payload
+        countries: action.payload,
       };
     case "SELECT_LANGUAGE":
       return {
         ...state,
-        language: action.payload
+        language: action.payload,
       };
     case "GET_ALL_FILTER_OPTIONS":
       return {
         ...state,
-        allFilterOptions: action.payload
+        allFilterOptions: action.payload,
       };
     case "GET_AIRTABLE_RECORDS": {
       const { pageStateEnum, records } = action.payload;
       const newState = { ...state };
       const pageState = newState[pageStateEnum as keyof State] as PageState;
       pageState.records = records;
       return newState;
-    };
+    }
     case "UPDATED_AT":
       return {
         ...state,
-        updatedAt: action.payload
+        updatedAt: action.payload,
       };
     case "UPDATE_FILTER": {
       const { pageStateEnum, filterType, filterValue } = action.payload;
       const newState = { ...state };
       const pageState = newState[pageStateEnum as keyof State] as PageState;
       pageState.filters[filterType as FilterType] = new Set(filterValue);
       return newState;
-    };
+    }
     // Note: deprecated but leaving here in case we ever need again
```

Comment:
on second thought what do we think about just deleting this lol. i think I made a bad call leaving here and putting a deprecated comment

Reply:
like it was never there!